### ISSUE TITLE
Units

### ISSUE THREAD

**Issue by Ephraim-Bryski** [stefnotch/quantum-sheet#46]
- [ ] SI units!
- [ ] Custom defined units
- [ ] Check out https://github.com/sharkdp/insect

**Comment by stefnotch** [comment]
Alternatively we could use the `[ ]` for units
https://math.stackexchange.com/questions/2332348/is-there-a-symbol-for-has-units-of/2332352#2332352

**Comment by Ephraim-Bryski** [comment]
> Alternatively we could use the `[ ]` for units
> https://math.stackexchange.com/questions/2332348/is-there-a-symbol-for-has-units-of/2332352#2332352

Yeah, I think that might be the most elegant. The only issue is if we decide to use brackets for arrays, it could get confusing.
